ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -128,11 +154,12 @@ const Post = ({ post }: { post: PostType }) => {
   );
 
   const isBreakingNews = pathOr(false, ['options', 'isBreakingNews'], post);
+  const timestamp = pathOr('', ['dates', 'curated'], post);
```

Comment:
Can we avoid use of `pathOr` in favour of optional chaining?

```suggestion
  const timestamp = post?.dates?.curated ?? '';
```

or is it 
```suggestion
  const timestamp = post?.dates?.curated || '';
```

(I can never remember the difference between ?? and ||)